templates/csharp/sso-tab/Components/Welcome.razor

```diff
@@ -22,25 +22,33 @@ else
             <h1 class="center">Congratulations@(!String.IsNullOrEmpty(userName) ? ", " + userName : "")!</h1>
             <p class="center">Your app is running in your @(GetEnvironmentName())</p>
             <FluentTabs style="width: 80%; justify-content: space-between; margin: 4rem auto;">
-                <FluentTab style="font-size: 14px; min-height: 32px;">1. Build your app locally</FluentTab>
-                <FluentTab>2. Provision and Deploy to the Cloud</FluentTab>
-                <FluentTab>3. Publish to Teams</FluentTab>
-                <FluentTabPanel>
-                    <div style="display:block; width:100%;">
-                        <EditCode />
-                        @if (isInTeams)
-                        {
-                            <CurrentUser UserName="@userName" />
-                        }
-                        <Graph ErrorMessage="@errorMessage"/>
-                    </div>
-                </FluentTabPanel>
-                <FluentTabPanel>
-                    <Deploy/>
-                </FluentTabPanel>
-                <FluentTabPanel>
-                    <Publish/>
-                </FluentTabPanel>
+                <FluentTab style="font-size: 14px; min-height: 32px;">
+                    <Header>1. Build your app locally</Header>
+                    <Content>
+                        <div style="display:block; width:100%;">
+                            <EditCode />
+                            @if (isInTeams)
+                            {
+                                <CurrentUser UserName="@userName" />
+                            }
+                            <Graph ErrorMessage="@errorMessage" />
+                        </div>
+                    </Content>
+                </FluentTab>
+                <FluentTab>
+                    <Header>2. Provision and Deploy to the Cloud</Header>
+                    <Content>
+                        <Deploy />
+                    </Content>
+                </FluentTab>
+                <FluentTab>
+                    <Header>
+                        3. Publish to Teams
+                    </Header>
```

Comment:
inconsistent code style. 
```
<Header> 3. Publish to Teams </Header>
```
or also adding new lines for other header.

Reply:
I'll standardize it.